_gsocproposals/2020/proposal_VecGeomGJK.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Implementation of Gilbert-Johnson-Keerthi Algorithm for Convex Shapes in VecGeom
+layout: gsoc_proposal
+project: VecGeom
+year: 2020
+organization:
+    - CERN
+---
+
+## Description
```

Comment:
What is the project? I know that it's in the title, but shouldn't it be explicitly said in the description?

Reply:
I don't know what you mean with this, both VecGeom and GJK are mentioned in the description.

Reply:
I changed the wording.